beacon-chain/sync/pending_attestations_queue.go

```diff
@@ -42,7 +42,7 @@ func (s *Service) processPendingAttsQueue() {
 // 1. Clean up invalid pending attestations from the queue.
 // 2. Check if pending attestations can be processed when the block has arrived.
 // 3. Request block from a random peer if unable to proceed step 2.
-func (s *Service) processPendingAtts(ctx context.Context) error {
+func (s *Service) processPendingAttsByBlkRoot(ctx context.Context) error {
 	ctx, span := trace.StartSpan(ctx, "processPendingAtts")
```

Comment:
Since we changed the function name, shouldn't we rename the span too?
```suggestion
func (s *Service) processPendingAttsByBlkRoot(ctx context.Context) error {
	ctx, span := trace.StartSpan(ctx, "processPendingAttsByBlkRoot")
```